rest/src/main/java/org/jboss/pnc/rest/endpoint/ProductVersionEndpoint.java

```diff
@@ -134,7 +134,8 @@ public Response getSpecific(
     @Path("/{id}")
     public Response update(@ApiParam(value = "Product Version id", required = true) @PathParam("id") Integer id,
             ProductVersionRest productVersionRest) throws RestValidationException {
-        return super.update(id, productVersionRest);
+        productVersionProvider.update(id, productVersionRest);
```

Comment:
You should not update the endpoint, but rather create a validator and bind it to the Rest entity.
See RepositoryConfigurationEndpoint#update RepositoryConfigurationRest.internalUrl ScmUrl ScmUrlValidator